experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -392,11 +395,23 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 		);
 	}
 
+	public getTreeStatus(): TreeStatus {
+		const path = this.cursor.getFieldPath().parent;
+		if (path === undefined) {
+			return TreeStatus.Deleted;
+		}
+		// TODO: This is a slow initial implementation which traverses the entire path up to the root of the tree.
```

Comment:
We arn't consistent about it, and don't document it anywhere, but one convention I like from my previous team it to make performance related TODOs more searchable like this:

```suggestion
		// TODO:Performance: This is a slow initial implementation which traverses the entire path up to the root of the tree.
```

Reply:
Also, I don't like code which contains TODO's being duplicated: better to have one call the other or have them both call a shared helper which contains the TODO to reduce TODOs in the code and make them easier to implement without missing any ussages.

Reply:
Also factoring this policy out into its own function will make testing the optimized version against it easier (the simple version can be moved to a test utility and used to test against if needed)

Reply:
updated TODO comment and created helper function to remove redundant TODO comment